stable/filebeat/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 description: A Helm chart to collect Kubernetes logs with filebeat
 icon: https://www.elastic.co/assets/blt47799dcdcf08438d/logo-elastic-beats-lt.svg
 name: filebeat
-version: 1.7.0
```

Comment:
What's the procedure here should we bump this to 2.0.0 since the major version of filebeat is increasing?

Reply:
Yeah, I was not sure about this one.

It's kind of a breaking change, but it's your call

Reply:
I would think we want to bump the major version.

Reply:
Done